js/main.js

```diff
@@ -25,9 +25,86 @@ function getRandomFractionNumber(min, max, numberSymbols) {
 
   const randomNumber = min + Math.random() * (max - min);
 
-  return Number(randomNumber.toFixed(1 * numberSymbols));
+  return Number(randomNumber.toFixed(numberSymbols));
 }
 
-getRandomFractionNumber(1, 10, 2);
+getRandomFractionNumber(1, 10, 0);
 
 
+// Возвращает фотографию пользователя
+const AVATAR_COUNTS = 10;
+
+function getAvatarUrl () {
+  const randomNumber = getRandomFractionNumber(1, AVATAR_COUNTS, 0);
+
+  let AvatarUrl = 'img/avatars/userget' + `0${ randomNumber }.png`;
+  if (randomNumber === 10) {
+    AvatarUrl = 'img/avatars/userget' + `${ randomNumber }.png`;
+  }
+
+  return AvatarUrl;
+}
+
+const avatarArray = [];
+
+function getavatarValue () {
+  let nextValue;
+  while (avatarArray.length < AVATAR_COUNTS) {
+    nextValue = getAvatarUrl();
+    if(!avatarArray.includes(nextValue))
+    {avatarArray.push(nextValue);
+      return nextValue;
+    }
+  }
+}
+
+const createAvatarArray = Array.from({length: 10}, getavatarValue);
+
+
+// возвращает время заезда и выезда
+function getCheckinAndCheckout () {
+  const randomNumber = getRandomFractionNumber(12, 14, 0);
+  const checkinAndCheckout = `${ randomNumber }` + ':00';
+  return checkinAndCheckout;
+}
+
+// Массивы с рандомными данными
+const title = ['Квартира в центре Питера', 'Квартира в центре Москвы', 'Квартира в центре Сочи'];
+const type = ['palace', 'flat', 'house', 'bungalow', 'hotel'];
+const features = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
+const description = ['Все отлично', 'Все плохо'];
+const photos = ['https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg'];
+const calculationLat = (Math.random() + 35).toFixed(5);
+const calculationLng = (Math.random() + 139).toFixed(5);
+
+// Возвращает случайные элементы массива
+const getRandomRiver = (arr) => arr.sort(() => 0.5 - Math.random()).slice(0, getRandomNumber(1, arr.length));
```

Comment:
Нужно придерживаться единого стиля определения функций, по критерию, если ты используешь где-то 
`function getCheckinAndCheckout` такой вид например то и тут должно быть `function getRandomRiver`
 или в предыдущих функциях замени на стрелочный вид
```suggestion
const getRandomRiver = (arr) => arr.sort(() => 0.5 - Math.random()).slice(0, getRandomNumber(1, arr.length));
```

Reply:
Исключением может быть только если мы используем функции как аргументы/коллбеки при именнованном определении функций или нам нужен контекст, а мы используем стрелочные, тогда наоборот. В общем так или иначе исключения могут быть, но не тут, старайся всегда функции определять одинаково, потом по критериям при проверке проект завернет автотест

Reply:
Название не соответствует контексту, getRandomRiver -- не совсем понятно при чем тут "случайная реке" нужно что-то вроде getRandomArrayItem или типо того